src/admin/content-block/components/ContentBlockPreview/ContentBlockPreview.tsx

```diff
@@ -52,6 +56,7 @@ const COMPONENT_PREVIEW_MAP = Object.freeze({
 	[ContentBlockType.IFrame]: BlockIFrame,
 	[ContentBlockType.Intro]: BlockIntro,
 	[ContentBlockType.Image]: BlockImage,
+	[ContentBlockType.MediaGrid]: BlockMediaListWrapper,
```

Comment:
Noemen we deze niet beter uniform:
MediaGrid => BlockMediaGridWrapper
of
MediaList => BlockMediaListWrapper

Reply:
Fixed